packages/components/pie-tag/test/component/pie-tag.spec.ts

```diff
@@ -25,99 +27,78 @@ const variantsToIsStrongStyle: Array<VariantToBgStyle> = [
     { variantName: 'brand-06', bgStyle: '--dt-color-support-brand-06' },
 ];
 
-const icon = '<svg slot="icon" data-test-id="tag-icon" xmlns="http://www.w3.org/2000/svg" role="presentation" focusable="false" fill="currentColor" viewBox="0 0 16 16" class="c-pieIcon c-pieIcon--plusCircle"><path d="M8.656 4.596H7.344v2.748H4.596v1.312h2.748v2.748h1.312V8.656h2.748V7.344H8.656V4.596Z"></path><path d="M12.795 3.205a6.781 6.781 0 1 0 0 9.625 6.79 6.79 0 0 0 0-9.625Zm-.927 8.662a5.469 5.469 0 1 1-7.734-7.735 5.469 5.469 0 0 1 7.734 7.736Z"></path></svg>';
-
 test.describe('PieTag - Component tests', () => {
-    test('should render successfully', async ({ mount, page }) => {
+    test('should be visible', async ({ page }) => {
         // Arrange
-        await mount(PieTag, {
-            slots: {
-                default: 'Label',
-            },
-        });
+        const tagPage = new BasePage(page, 'tag');
+        await tagPage.load();
 
         // Act
-        const tag = page.locator(componentSelector);
+        const tagComponent = page.locator(tag.selectors.container.dataTestId, { hasText: 'Label' });
 
         // Assert
-        await expect(tag).toBeVisible();
+        await expect(tagComponent).toBeVisible();
     });
 
     test.describe('icon slot', () => {
-        test.describe('when passed', () => {
-            test.describe('if the size is large', () => {
-                test('should render the icon', async ({ mount, page }) => {
-                    // Arrange
-                    await mount(PieTag, {
-                        slots: {
-                            default: 'Label',
-                            icon,
-                        },
-                    });
-
-                    // Act
-                    const tagIcon = page.locator(tagIconSelector);
-
-                    // Assert
-                    await expect(tagIcon).toBeVisible();
-                });
-            });
-
-            test.describe('if the size is small', () => {
-                test('should NOT render the icon', async ({ mount, page }) => {
-                    // Arrange
-                    await mount(PieTag, {
-                        props: {
-                            size: 'small',
-                        },
-                        slots: {
-                            default: 'Label',
-                            icon,
-                        },
-                    });
-
-                    // Act
-                    const tagIcon = page.locator(tagIconSelector);
-
-                    // Assert
-                    await expect(tagIcon).toBeHidden();
-                });
-            });
+        test('should render icon when size is large', async ({ page }) => {
+            // Arrange
+            const tagPage = new BasePage(page, 'tag--default-with-icon');
+            await tagPage.load();
```

Comment:
Even though `large` is the [default size](https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-tag/src/defs.ts#L47), can you be explicit and pass it as the prop value, just so this test won't fail should the default value ever change.